docs/source/get_started/includes/quickstart.yaml

```diff
@@ -1,44 +1,43 @@
 version: v0.1
-
-listen:
-  address: 0.0.0.0 # or 127.0.0.1
-  port: 10000
-  # Defines how Arch should parse the content from application/json or text/pain Content-type in the http request
+listener:
+  address: 127.0.0.1
+  port: 8080 #If you configure port 443, you'll need to update the listener with tls_certificates
   message_format: huggingface
 
 # Centralized way to manage LLMs, manage keys, retry logic, failover and limits in a central way
 llm_providers:
   - name: OpenAI
     provider: openai
     access_key: $OPENAI_API_KEY
-    model: gpt-4o
+    model: gpt-3.5-turbo
     default: true
-    stream: true
 
 # default system prompt used by all prompt targets
-system_prompt: You are a network assistant that just offers facts; not advice on manufacturers or purchasing decisions.
+system_prompt: |
+  You are a network assistant that helps operators with a better understanding of network traffic flow and perform actions on networking operations. No advice on manufacturers or purchasing decisions.
 
 prompt_targets:
   - name: reboot_devices
-    description: Reboot specific devices or device groups
-
-    path: /agent/device_reboot
+    description: Reboot a list of devices
+    endpoint:
+      name: app_server
+      path: /agent/device_reboot
     parameters:
       - name: device_ids
         type: list
-        description: A list of device identifiers (IDs) to reboot.
-        required: false
-      - name: device_group
-        type: str
-        description: The name of the device group to reboot
-        required: false
+        description: A list of device identifiers (IDs).
+        required: true
+      - name: days
+        type: int
+        description: A list of device identifiers (IDs)
+        default: "7"
```

Comment:
I think this getting started example isn't great. We can do better. We can have following endpoint,

```
 description: Get device metadata that contains details like sku, up time, num cpus/gpus etc.
    endpoint:
      name: metadata_server
      path: /v1/metadata
    parameters:
      - name: device_ids
        type: list
        description: A list of device identifiers (IDs).
        required: true
```

Reply:
Fair. In this instance I am trying to match our README with docs. I'll open an another issue to improve getting started